pkgs/tools/misc/betterdiscordctl/default.nix

```diff
@@ -2,13 +2,13 @@
 
 stdenv.mkDerivation rec {
```

Comment:
Would love to have this to be using `stdenvNoCC`
```suggestion
stdenvNoCC.mkDerivation rec {
```

Reply:
Please create a PR with that change and this update.

Reply:
Done, see #129985.